OrbitLinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -41,6 +41,47 @@ void UprobesUnwindingVisitor::visit(SamplePerfEvent* event) {
   listener_->OnCallstack(returned_callstack);
 }
 
+void UprobesUnwindingVisitor::visit(SampleFPPerfEvent* event) {
+  CHECK(listener_ != nullptr);
+
+  if (current_maps_ == nullptr) {
+    return;
+  }
+
+  /* TODO(kuebler): handle the mapping/patching for uprobes
```

Comment:
Are the comments really needed?

Reply:
We should not commit uncommented code, so IMO this all should be removed.

Reply:
@ronaldfw , I guess you mean "we should not commit *commented code", well I hope ;-)

Reply:
Yeah, I guess my wording was bad here. "We should not commit code that is commented out."

Reply:
Totally agree. Thanks for spotting this. I removed the comment.